transformers/component_tests/jug/transformer_test.go

```diff
@@ -36,18 +39,18 @@ import (
 var _ = Describe("Executing the transformer", func() {
```

Comment:
🤔 I wonder if we want to name this `configured_transformer_test`. Still getting used to `converter => transformer`, which I think is a win - but the various vat "transformer tests" threw me off for a minute